app/views/publish_document/published.html.erb

```diff
@@ -1,4 +1,9 @@
-<% content_for :back_link, document_path(@document) %>
+<% if @document.review_state == "reviewed" %>
+  <% content_for :back_link, t("application.back_link.back"), documents_path %>
```

Comment:
I don't think this will work

Reply:
Do you mean the implementation or because my change to the component hasn't been merged?

Reply:
it looks like we're passing three arguments to `content_for` when it can just accept two: the key and the value. The value of this is then passed to the component as the href argument: https://github.com/alphagov/content-publisher/blob/6ec7337fea74534cce1ea515633a72981cd936c3/app/views/layouts/application.html.erb#L44